src/sentry/testutils/helpers/task_runner.py

```diff
@@ -5,7 +5,7 @@
 from celery import current_app
 from contextlib import contextmanager
 from django.conf import settings
-from mock import patch
+from sentry.utils.compat.mock import patch
```

Comment:
Is the idea that we'd remove this compat thing in another pr?

Reply:
Yeah, it'd be a lot more modifications and I wanted to keep a followup small.